core/services/ocr2/plugins/ocr2keeper/evmregistry/v21/logprovider/buffer.go

```diff
@@ -15,10 +15,10 @@ import (
 )
 
 var (
-	// maxLogsPerUpkeepInBlock is the maximum number of logs allowed per upkeep in a block.
-	maxLogsPerUpkeepInBlock = 32
-	// maxLogsPerBlock is the maximum number of blocks in the buffer.
-	maxLogsPerBlock = 1024
+	// defaultFastExecLogsHigh is the default upper bound / maximum number of logs that Automation is committed to process for each upkeep,
+	// based on available capacity, i.e. if there are no logs from other upkeeps.
+	// Used by Log buffer to limit the number of logs we are saving in memory for each upkeep in a block
+	defaultFastExecLogsHigh = 10
```

Comment:
I'm not sure what the significance of fastExec is here; going by the comment block would it make sense to just call this `maxLogsPerUpkeep`? Or is that too broad?

Reply:
I wanted to keep our terminology stable cross components, for each component it might be represented with a better name but then we have multiple names for the same variable.

See AUTO-7207 for more information

Reply:
Do we want to keep the overall limit in the buffer (`maxLogsPerBlock` before), `NumOfLogUpkeeps * FastExecLogsHigh` in ticket?

Reply:
Is the number of upkeeps a hard cap? What do we expect if upkeeps go bigger than that cap, would we want the log buffer to increase indefinitely in size?

Reply:
I've reverted this part, means we will have hard limit based on the number of log upkeeps specified in the config